crypto/sha/sha3.c

```diff
@@ -8,13 +8,15 @@
  */
 
 #include <string.h>
+#include "crypto/s390x_arch.h"
```

Comment:
This needs to go into a `#if` so that it is only included on the s390x platform.